### PR TITLE
Material UI

### PR DESCRIPTION
**Pull request by speshori**
Have most of the pages up to date with styling, and had skiball merged into this branch as well. Should have done that pull request separately, sorry about that, but just happened in the process of my work.

### CODE REVIEW

File: server/routes/coinflip.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def coinflip():
 
         if amount > 0:
             if did_win:
-                tickets_won = amount
+                tickets_won = amount*2
```
Comment (dvkm) [inline server/routes/coinflip.py]: Why this change?

File: server/routes/skiball.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+        transaction = Transaction(
+            user_id=user_id, ticket_amount=amount, activity="skiball"
+        )
+        fee = Transaction(user_id=user_id, ticket_amount=-30, activity="skiball")
+
+        db.session.add(fee)
+        db.session.commit()
+        db.session.add(transaction)
+        db.session.commit()
```
Comment (ak2253) [inline server/routes/skiball.py]: I think you can get away with deleting db.session.commit() on line 42.

Comment (ak2253) [reply server/routes/skiball.py]: Because the one line 44 should commit both table actions.

**Review by ak2253 — APPROVED** [review]

File: server/routes/coinflip.py (hunk 1 above)
Comment (speshori) [reply server/routes/coinflip.py]: That was the way the original game was said out to be in the writeup so i notieced it wasn't there and changed it.

File: server/routes/skiball.py (hunk 2 above)
Comment (speshori) [reply server/routes/skiball.py]: Okay got it.

File: server/routes/coinflip.py (hunk 1 above)
Comment (dvkm) [reply server/routes/coinflip.py]: That was the case if playing coinflip takes away the point in the first place. If you want this change, you'll have to change it so that the tickets is deducted every game we play. However, that would be an extra transaction every game and that might be confusing. I say we just remove it.

**Review by dvkm — CHANGES_REQUESTED** [review]
change back tickets_won thing.

**Review by dvkm — APPROVED** [review]
